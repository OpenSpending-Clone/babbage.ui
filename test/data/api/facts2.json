{
  "fields": [
    "approved",
    "adjusted",
    "executed",
    "classification_administrative_admin1.admin1",
    "datetime.year"
  ],
  "cell": [
  ],
  "page": 1,
  "status": "ok",
  "total_fact_count": 190525,
  "order": [
  ],
  "data": [
    {
      "executed": 0.0,
      "classification_administrative_admin1.admin1": "Central",
      "approved": 100,
      "adjusted": 200,
      "datetime.year": 2014
    },
    {
      "executed": 5,
      "classification_administrative_admin1.admin1": "Central",
      "approved": 10,
      "adjusted": 20,
      "datetime.year": 2014
    },
    {
      "executed": 10,
      "classification_administrative_admin1.admin1": "Central",
      "approved": 20,
      "adjusted": 0,
      "datetime.year": 2014
    },
    {
      "executed": 0.0,
      "classification_administrative_admin1.admin1": "Central",
      "approved": 0,
      "adjusted": 0,
      "datetime.year": 2014
    },
    {
      "executed": 1,
      "classification_administrative_admin1.admin1": "Other",
      "approved": 2,
      "adjusted": 1,
      "datetime.year": 2014
    }
  ],
  "page_size": 5
}