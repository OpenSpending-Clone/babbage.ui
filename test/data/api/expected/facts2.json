{
  "headers": [
    {
      "key": "approved",
      "label": "approved",
      "numeric": true
    },
    {
      "key": "adjusted",
      "label": "adjusted",
      "numeric": true
    },
    {
      "key": "executed",
      "label": "executed",
      "numeric": true
    },
    {
      "key": "classification_administrative_admin1.admin1",
      "label": "admin1",
      "numeric": false
    },
    {
      "key": "datetime.year",
      "label": "year",
      "numeric": false
    }
  ],
  "columns": [
    [
      100,
      200,
      0,
      "Central",
      2014
    ],
    [
      10,
      20,
      5,
      "Central",
      2014
    ],
    [
      20,
      0,
      10,
      "Central",
      2014
    ],
    [
      0,
      0,
      0,
      "Central",
      2014
    ],
    [
      2,
      1,
      1,
      "Other",
      2014
    ]
  ],
  "info": {
    "total": 190525,
    "pageSize": 5,
    "page": 1
  }
}